### ISSUE TITLE
Work out why tag build ignores test errors

### ISSUE THREAD

**Issue by l0b0** [linz/linz-utils-perl#45]
See https://github.com/linz/linz-utils-perl/actions/runs/1547465737.

![Screenshot from 2021-12-08 09-49-26](https://user-images.githubusercontent.com/168301/145104007-d59fd748-c8d3-4170-a373-c79efa4a69b2.png)

**Comment by billgeo** [comment]
Is this something we need to fix before releasing in #73?

**Comment by l0b0** [comment]
No. It's disconcerting that we have these errors at all, but we don't know why they are there and it's possible they are a complete red herring.
